data/json/items/gun/9mm.json

```
@@ -1,4 +1,27 @@
 [
+  {
+    "id": "90two",
+    "copy-from": "pistol_base",
+    "looks_like": "glock_17",
+    "type": "GUN",
+    "name": { "str": "Beretta 90two" },
```

```suggestion
    "name": { "str": "Beretta 90-two" },
```
[The operator's manual](https://web.archive.org/web/20160304030030/http://www.beretta90two.com/dati/ContentManager/files/manuale/90%20two.pdf) has a hyphen there